akka-http2-support/src/test/scala/akka/http/impl/engine/http2/Http2FrameHpackSupport.scala

```diff
@@ -0,0 +1,100 @@
+package akka.http.impl.engine.http2
+
+import java.io.{ ByteArrayInputStream, ByteArrayOutputStream }
+
+import akka.http.impl.util.StringRendering
+import akka.http.scaladsl.model.{ ContentType, HttpEntity, HttpHeader, HttpRequest, HttpResponse }
+import akka.http.scaladsl.model.headers.RawHeader
+import akka.stream.Materializer
+import akka.stream.scaladsl.Source
+import akka.util.ByteString
+import com.twitter.hpack.{ Decoder, Encoder, HeaderListener }
+import org.scalatest.concurrent.ScalaFutures
+
+import scala.collection.immutable.VectorBuilder
+
+/** Helper that allows automatic HPACK encoding/decoding for wire sends / expectations */
+trait Http2FrameHpackSupport extends Http2FrameProbeDelegator with Http2FrameSending with ScalaFutures {
```

Comment:
Moved here without significant changes from `AutomaticHpackWireSupport` in `Http2ServerSpec`